modules/suitesparse/metadata.json

```diff
@@ -14,7 +14,9 @@
     "versions": [
         "7.6.0",
         "7.6.0.bcr.1",
-        "7.10.1"
+        "7.10.1.bcr.1"
     ],
-    "yanked_versions": {}
+    "yanked_versions": [
```

Comment:
This should be a map, with the value being the reason to yank and recommended fix

Reply:
update metadata at https://github.com/bazelbuild/bazel-central-registry/pull/4344/commits/df337a520764ed700c55ec002ad80be61159fb7a.